src/gigs-board/pages/community/Sidebar.jsx

```diff
@@ -0,0 +1,368 @@
+    ))}
+  </div>
+);
+
+const Card = ({ content }) => (
+  <div
+    className="card"
+    style={{
+      border: "none",
+    }}
+  >
+    <div className="card-body">
+      <p className="card-text">{content}</p>
+    </div>
+  </div>
+);
+
+const CommunityOverview = (
+  <Card title={`${community.title} Overview`} content={CommunitySummary}></Card>
+);
+
+const TeamMember = ({ member }) => (
+  <div className="d-flex align-items-start mb-1 justify-content-start">
+    <img
+      src={member.avatar}
+      alt={member.id}
+      style={{
+        width: "50px",
+        height: "50px",
+        borderRadius: "50%",
+        marginRight: "10px",
+      }}
+    />
+    <div style={{ paddingTop: "7px" }}>
+      <h6 style={{ fontSize: "110%", fontWeight: "bold", marginBottom: "0" }}>
+        {member.id}
+      </h6>
+      <p style={{ fontSize: "100%" }}>{member.wallet}</p>
+    </div>
+  </div>
+);
+
+// Team Members List
+const TeamMembersList = mockTeamMembers.map((member) => (
+  <TeamMember key={member.id} member={member} />
+));
+
+// More Info Button
+const MoreInfoButton = (
+  <div className="d-flex justify-content-start">
+    <button
+      type="button"
+      class="btn"
+      style={{
+        color: "black",
+        fontSize: "0.8rem",
+        fontWeight: "bold",
+      }}
+      href="#"
+    >
+      See More
+    </button>
+  </div>
+);
+
+// Team Card
+const TeamsCard = (
+  <div style={{ width: "100%" }}>
+    <Card
+      title={"Team Members"}
+      content={
+        <div>
+          <h5
+            style={{
+              fontSize: "1.3rem",
+              fontWeight: "800",
+              marginBottom: "20px",
+            }}
+          >
+            Group Moderators
+          </h5>
+          {TeamMembersList}
+          {MoreInfoButton}
+        </div>
+      }
+    ></Card>
+  </div>
+);
+
+// Define the Sidebar component
+const Sidebar = () => (
+  <div class="col-md-12 d-flex flex-column align-items-end">
+    {CommunityOverview}
+    <hr style={{ width: "100%", borderTop: "4px solid #00000033" }} />
+    {TeamsCard}
+  </div>
+);
+
+// Define a wrapper component for the content to the left of the sidebar
+const Content = () => (
+  <div class="col-md-0">
+    {/* 
+      Place the components or content you want to display to the left of 
+      the sidebar here.
+    */}
+  </div>
+);
+
+// Define the layout for the page
+const PageLayout = () => (
+  <div class="container">
+    <div class="row">
+      <Content />
+      <Sidebar />
+    </div>
+  </div>
+);
+
+return <PageLayout />;
+// END OF SIDEBAR
+
+// USAGE
+// copy and paste this into the return function of a page you want the sidebar on
+// make sure to add `col-mb-8` to the class of the div you want to
+// take up 2/3 of the screen to the left
+
+// then make sure to add `col-mb-4` to the class of the div you want to
+// take up 1/3 of the screen to the
+
+{
+  /* <div class="col-4 container-fluid">
```

Comment:
A better pattern is Makes Content as a prop of function PageLayout. Refer to src/gigs-board/components/layout/Page.jsx as example.